spec/models/organization_webhook_spec.rb

```diff
@@ -18,18 +18,48 @@
   it { should validate_presence_of(:github_organization_id) }
   it { should validate_uniqueness_of(:github_organization_id) }
 
-  describe "#github_organization" do
-    it "requests a GitHubOrganization" do
-      expect_any_instance_of(Organization).to receive(:github_organization)
-      subject.github_organization
+  describe "#admin_org_hook_scoped_github_client" do
+    context "token is present" do
+      before do
+        allow(subject).to receive_message_chain(:users, :first, :token) { "token" }
+      end
+
+      it "raises a NoValidTokenError" do
```

Comment:
We don't expect to raise an error here, just need to update test name?

Reply:
Good catch 👍 a case of some copy pasta 🍝